shed/tests/test_streams.py

```diff
@@ -546,6 +549,101 @@ def f(img1):
         assert n in assert_docs
 
 
+def test_filter_descriptor(exp_db, start_uid1):
+    source = Stream()
```

Comment:
I would personally add tests that generate data explicitly with your `to_event_model` function. So we can separate the databroker stuff from these simple tests (and test `exp_db`'s interaction elsewhere).
This would remove all dependencies on `exp_db` and `start_uid1` which would make debugging just the simple stream functionality easier (i.e. dont need to set up a mock db).

Reply:
PRs welcome!